src/command/config/profile/mod.rs

```diff
@@ -0,0 +1,42 @@
+mod auth;
+mod delete;
+mod list;
+mod show;
+
+use anyhow::Result;
+use serde::Serialize;
+use structopt::StructOpt;
+
+use crate::command::RoverStdout;
+
+#[derive(Debug, Serialize, StructOpt)]
+pub struct Profile {
+    #[structopt(subcommand)]
+    command: Command,
+}
+
+impl Profile {
```

Comment:
Is there a docs page for `profile` as well? I see that this PR changes the `authentication` and the `config` pages, but not `profile`. Should we update this as well?